swift/schema.py

```diff
@@ -77,7 +77,13 @@ class AstNode(Locatable):
 @ql.hideable
 class Type(Element):
     name: string
-    canonical_type: "Type"
+    canonical_type: "Type" | desc("""
+        This is the unique type we get after resolving aliases and desugaring. For example, given
+        ```
+        typealias MyInt == Int
```

Comment:
I don't think `==` is valid Swift, is it?

Reply:
Whoops...

Reply:
Fixed.